extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -566,6 +580,7 @@ export class AmpStoryStoreService {
       [StateProperty.PAUSED_STATE]: false,
       [StateProperty.RTL_STATE]: false,
       [StateProperty.SHARE_MENU_STATE]: false,
+      [StateProperty.SHOPPING_STATE]: {},
```

Comment:
Please update this to `SHOPPING_DATA` so the naming convention is equivalent to `ADD_SHOPPING_DATA`.